site/docs/components/overlay/accessibility.mdx

```diff
@@ -0,0 +1,44 @@
+---
+# Leave the frontmatter as is
+title:
+  $ref: ./#/title
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+### Best practices
```

Comment:
These don't sound like Best practices, but more like Interaction

Reply:
👍🏻 updated the title to say `Interactions` instead